source/_docs/automation/trigger.markdown

```diff
@@ -132,6 +132,12 @@ automation:
 
 The `for` template(s) will be evaluated when an entity changes as specified.
 
+<div class='note warning'>
+  
+  Template support to template trigger's `for` option was added in 0.96 and won't work in previous versions.
```

Comment:
Those kind of information are usually covered in the release notes. 0.96 was released almost three months ago and I doubt that using `for` on pre-0.96 releases is an issue of a large audience.

Reply:
@fabaff fair enough 🙂